llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -8842,6 +8843,8 @@ LoopVectorizationPlanner::tryToBuildVPlanWithVPRecipes(VFRange &Range) {
           J++;
         }
         MemberR->eraseFromParent();
+        if (!Member->getType()->isVoidTy())
```

Comment:
nit: could this be folded within the same `if` above?

Reply:
Yes, updated to take the name of MemberR